python/pyarrow/_parquet.pyx

```diff
@@ -75,6 +77,13 @@ cdef class Statistics(_Weakrefable):
                                         self.converted_type)
 
     def to_dict(self):
+        """Get dictionary represenation of statistics.
```

Comment:
```suggestion
        """
        Get dictionary represenation of statistics.
```

Another style nit .., but let's maybe try to do this consistently. I think we mostly start on the next line for multi-line docstring, and below in this diff it's a mix of both.